src/functional/ffront/common_types.py

```diff
@@ -10,6 +12,7 @@ class ScalarKind(IntEnum):
     BOOL = 1
     INT32 = 32
     INT64 = 64
+    INT = INT64 if np.int_ == np.int64 else INT32
```

Comment:
I realized it makes more sense to set the default to INT64 and only define INT as INT32 in case numpy does the same (windows platform).

Reply:
You mean you just "flip" the if-else clauses? Yeah, maybe slightly better, but it should be only two options, right?

Reply:
Yes, exactly.